src/main/containers/SourcePanel.jsx

```diff
@@ -4,10 +4,7 @@ import RequestBar from '../components/RequestBar.jsx';
 import { smallData } from '../dummyData';
 
 const SourcePanel = (props) => {
-  const [isLoading, setLoading] = useState(false);
-  const [data, setData] = useState(undefined);
-
-  const { treeCount, updateTreeCount } = props;
+  const { treeCount, updateTreeCount, data, setData } = props;
```

Comment:
similar to the above comment, lets put this on new line. will make the linter happy. 
 const {
    treeCount, updateTreeCount, data, setData,
  } = props;

Reply:
I fixed this manually, was blocking other features